opal/mca/accelerator/rocm/accelerator_rocm_module.c

```diff
@@ -123,6 +147,10 @@ static int mca_accelerator_rocm_check_addr (const void *addr, int *dev_id, uint6
             opal_accelerator_rocm_lazy_init();
             *dev_id = srcAttr.device;
             ret = 1;
+            // TODO: on Frontier the host can access any device memory
+            // is that true everywhere?
+            //*flags |= MCA_ACCELERATOR_FLAGS_UNIFIED_MEMORY;
+            //*dev_id = srcAttr.device;
```

Comment:
please remove the four lines that have been commented out. Regarding the question: yes, ping me in slack if you want more details :-)

Reply:
Done